skops/card/_model_card.py

```diff
@@ -1315,3 +1315,59 @@ def render(self) -> str:
             sections inserted.
         """
         return "\n".join(self._generate_card())
+
+    def _iterate_key_section_content(
```

Comment:
I wonder if this should be a local func inside `get_toc`

Reply:
I think it's better to have it as a private method, because I can imagine other uses for it than just for `get_toc`.